include/fiction/technology/charge_distribution_surface.hpp

```diff
@@ -391,7 +391,25 @@ class charge_distribution_surface<Lyt, false> : public Lyt
         return strg->dist_mat[index1][index2];
     }
     /**
-     * Calculates and returns the electrostatic potential between two cells.
+     * Calculates and returns the chargeless electrostatic potential between two cells.
+     *
```

Comment:
Could you add a comment to use the other function if the electrostatic potential is required?